pkg/destroy/openstack/openstack.go

```diff
@@ -103,7 +103,6 @@ func (o *ClusterUninstaller) Run() (*types.ClusterQuota, error) {
 		"deleteServerGroups":    deleteServerGroups,
 		"deleteTrunks":          deleteTrunks,
 		"deleteLoadBalancers":   deleteLoadBalancers,
-		"deletePorts":           deletePorts,
```

Comment:
I don't think we're guaranteed that all ports we need to delete are on a tagged networks. If nodes have additional user-specified networks, then these might not be tagged, yet we still need to delete these ports. Will Nova take care of that? I doubt it, as MAPO/CAPO creates the ports itself.

Reply:
Yes, I would think so.

Reply:
+1 and we would need to ignore not found errors, in case both goroutines try to delete same Port

Reply:
Hm, true, seems like 404 on DELETE is not ignored now. But from the other hand it's not a fatal issue, on the next port list we'll just go through this.